packages/beacon-node/test/e2e/network/mdns.test.ts

```diff
@@ -50,12 +49,7 @@ describe("mdns", function () {
       bootMultiaddrs: [],
       localMultiaddrs: [],
       discv5FirstQueryDelayMs: 0,
-      discv5: {
```

Comment:
@wemeetagain do you know if this `discv5` is needed for this test? Setting it as `null` even cause the issue happen consistently in my environment, looks like that's really an issue with this `mdns` test, not just `timeout` issue

Reply:
I think we use the existence of discv5 to determine whether or not to initialize the peer discovery module in the peer manager